DEVELOPMENT.md

```diff
@@ -1,61 +1,30 @@
 # Development
 
-## Prerequisites
+## Setup
 
-In order to build Eunomia, you'll need the following:
+Take a look at the [Prerequisites](./GETTING_STARTED.md#Prerequisites) section in the Getting started guide.
 
-- [Git](https://git-scm.com/downloads)
-- [Go 1.12+](https://golang.org/dl/)
-- [Docker](https://docs.docker.com/install/)
-- [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
-- Access to a Kubernetes cluster
-  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (optional)
-  - [Minishift](https://www.okd.io/minishift/) (optional)
-
-### Checkout your fork
-
-To check out this repository:
-
-1. Create a [fork](https://help.github.com/en/articles/fork-a-repo) of this repo
-2. Create the directories and clone your fork
-
-```
-mkdir -p $GOPATH/src/github.com/KohlsTechnology
-cd $GOPATH/src/github.com/KohlsTechnology
-git clone https://github.com/<YOUR FORK>/eunomia.git
-cd eunomia
-```
+## Running Locally for Development Purposes
 
-### Installing on a Mac
+The most efficient way to develop the operator locally is run the code on your local machine. This allows you to test code changes as you make them.
 
-All the components can easily be installed via [Homebrew](https://brew.sh/) on a Mac:
+To achieve this, run the helper script:
 
 ```shell
```

Comment:
What platforms are supported? This will obviously won't work on a Windows machine,